README.md

```diff
@@ -294,7 +294,13 @@ The example below gives AAPL daily OHLCV data in a DataFrame format.
 import alpaca_trade_api as tradeapi
 
 api = tradeapi.REST()
+# all of these examples work
 aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from='2019-01-01', to='2019-02-01').df
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01'), to='2019-01-10').df
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01').date(), to='2019-01-10').df
```

Comment:
As an example/document, it's a bit confusing to put this conversion? How about `datetime.date(2019, 1, 1)`?

I couldn't tell what type this is `date()` was returning but maybe only me.